utils/HelpAbstractionGenerator/StaticUtilities.cs

```diff
@@ -38,13 +38,18 @@ public static string wrapContentIntoHtmlDocument(List<HAP.HtmlNode> bodyContent)
 
     public static class urlHelpers
     {
+        public static string year = "";
         public static string getHelpURL(string guid)
         {
             return getDefaultHelpURL() + "?guid=GUID-" + guid;
         }
         public static string getDefaultHelpURL()
         {
-            return "https://help.autodesk.com/view/OARX/2021/" + System.Globalization.CultureInfo.CurrentCulture.ThreeLetterWindowsLanguageName + "/";
+            if (year == "")
+                year = Microsoft.VisualBasic.Interaction.InputBox("This should be the most current version of the Autodesk documentation and may not be the current year shown", "Enter 4 digit year", DateTime.Now.Year.ToString()).Trim();
+            if (year == "")
```

Comment:
how about string.IsNullOrWhiteSpace(year)

Reply:
It can't hurt, but I would note that line 41 is its definition, it gets initialized to an empty string and is only ever set inside  this getDefaultHelpURL() function. So, its technically fine as is.